### ISSUE TITLE
Cover Block: fixed background image fails a11y test

### ISSUE THREAD

**Issue by kstarcher** [WordPress/gutenberg#49902]
### Description

This is related to https://github.com/WordPress/gutenberg/issues/30505 which was fixed and closed, but a slightly different issue.

When a regular background image is used in a cover block, the alt text is correctly added and displayed in an img tag. However, if the "Fixed background" or "Repeated background" options are checked, the img tag switches to a div with role="img". SiteImprove then flags it as not having alt text, because an element with role="img" is supposed to have alt text. This happens regardless of whether or not the image has alt text in the media library.

### Step-by-step reproduction instructions

1. Add a cover image block and give it an image background
2. Under "Media Settings", give it alt text
3. Flip the "Fixed Background" switch on
4. Save and view, then run SiteImprove to test for accessibility

### Screenshots, screen recording, code snippet

<img width="1917" alt="Screenshot 2023-04-18 at 10 37 15 AM" src="https://user-images.githubusercontent.com/16195879/232859909-ada869d2-948a-496e-8876-a8717b51b72c.png">
<img width="1713" alt="Screenshot 2023-04-18 at 10 38 25 AM" src="https://user-images.githubusercontent.com/16195879/232859928-d30e62e2-aa46-4cbb-80b8-0812fc0f851c.png">

### Environment info

WP 6.2 using a custom theme. Gutenberg not installed, just using block editor.
Mac Chrome with SiteImprove extension. 

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by cngodles** [comment]
True. If the `<div>` element has the attribute/value pair of `role="img"`, it either needs an `aria-label` or a `role` to be set to `presentation`. This is a perfect example of automated testing catching this condition.

**Comment by joedolson** [comment]
This should be able to cover both cases: an image that is presentational or an image that contains content. I don't think users should generally add content images into a fixed background image, but...they probably will, so we might as well support that accessibly.

This means that the fixed cover background should support an alt text field. If the alt text field has content, then the div should have a role of img and an aria-label containing the alt text.

If the alt text does not have content, then the `role=img`should be omitted. `Role=presentation` is not required, as that's the native role for a `div` element.

**Comment by carolinan** [comment]
The block already has an alt text input field when an image is selected, so we should ensure it is not hidden when the fixed or repeated options are enabled.

**Comment by michaelbourne** [comment]
Good to note that the alt text field does disappear if you select the fixed image mode on the cover block. So either that needs to remain, or we opt for the "always presentation" images angle here.
